apps/docs/content/icons/getting-started/installation.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: Installation
+description: Getting started with Workleap Design Icons
+order: 1
+---
+
+export const installMethods = [
+  { title: "pnpm" },
+  { title: "yarn" },
+  { title: "npm" }
+];
+
+# Installation
+
+Using your package manager of choice, run the following command to install the icons package.
+
+## React Components
+
+If you are using React, you can install the React components package.
+
+<Tabs tabs={installMethods}>
```

Comment:
Can you create a wrapper component like this: 

Usage:
```tsx
<PackageManagerInstallation packages={["@hopper-ui/icons"]} />;
```

PackageManagerInstallation 's code
```tsx

type PackageManagerType = "pnpm" | "yarn" | "npm";

const DefaultPackageManagers: PackageManagerType[] = ["pnpm", "yarn", "npm"];

interface PackageManagerInstallationProps {
    packages: string[];
    packageManagers?: string[];
}

function PackageManagerInstallation({ packages, packageManagers = DefaultPackageManagers }: PackageManagerInstallationProps) {
    <Tabs tabs={packageManagers.map(x => ({ title: x }))}>
        <div>
    ```shell
    pnpm add {packages.join(" ")}
    ```
        </div>
        <div>
    ```shell
    yarn add -D {packages.join(" ")}
    ```
        </div>
        <div>
    ```shell
    npm add -D {packages.join(" ")}
    ```
        </div>
    </Tabs>;
}
```

Reply:
Si tu peux ajouté le svg du package manager ça serait cute aussi 

![image](https://github.com/gsoft-inc/wl-hopper/assets/38871812/a9a23871-7751-4a9f-b670-c5749395cb7d)